releases/3.0.18.md

```diff
@@ -0,0 +1,17 @@
+---
+title: TiDB v3.0.18 Release Notes
+category: Releases
+aliases: ['/docs/dev/releases/v3.0.18/']
+---
+
+# TiDB v3.0.18 Release Notes
+
+Release date: August 17, 2020
+
+TiDB version: v3.0.18
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix wrong hash key for decimal [#19185](https://github.com/pingcap/tidb/pull/19185)
```

Comment:
Release date is August 21.

TiKV 
Bug Fix: 
Change GC failure log to Warning level [#8444](https://github.com/tikv/tikv/pull/8444)

Reply:
TiDB bug fix:
Fix an encode bug that causing the wrong result of hashJoin with set and enum https://github.com/pingcap/tidb/pull/19172